BGAnimations/ScreenSystemLayer overlay.lua

```diff
@@ -243,7 +243,7 @@ t[#t+1] = RequestResponseActor("PingLauncher", 10)..{
 			callback=function(res, args)

```

Comment:
This should also handle the nil value, because otherwise we enable launcher functionality when the launcher is not running.

Reply:
AH good catch!! Done!